edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/IEdgeHub.cs

```diff
@@ -17,6 +17,8 @@ namespace Microsoft.Azure.Devices.Edge.Hub.Core
     /// </summary>
     public interface IEdgeHub : IDisposable
     {
+        string GetEdgeDeviceId();
```

Comment:
This is a const injected into the container.. why do we need this?

Reply:
Previously this was used in the middleware, and we leveraged the appbuilder to pass in custom params.  But now it's used in the TwinsController, and I don't see a good way to pass the param from the DependencyManager into the controller instance.